apps/web/pages/auth/login.tsx

```diff
@@ -58,13 +58,9 @@ export default function Login({
   };
 
   const telemetry = useTelemetry();
-
-  let callbackUrl = typeof router.query?.callbackUrl === "string" ? router.query.callbackUrl : "/";
-
-  // If not absolute URL, make it absolute
-  if (!/^https?:\/\//.test(callbackUrl)) {
-    callbackUrl = `${WEBSITE_URL}/${callbackUrl}`;
-  }
```

Comment:
This will reintroduce a previously fixed vulnerability

Reply:
I think it's the other way and this reintroduced it.

Previously with only the ternary operator, if router.query.callbackUrl, was a string, it assigned it after the WEBSITE_URL with interpolation, if it was not a string, it just assigned an empty one.

Now if you pass a malicious https:// url, the if regex test never kicks in, and the callbackUrl doesn't get replaced.

I think anyways?